utils/src/main/java/software/amazon/awssdk/utils/http/SdkHttpUtils.java

```diff
@@ -297,4 +307,12 @@ public static String appendUri(String baseUri, String path) {
     public static Optional<String> firstMatchingHeader(Map<String, List<String>> headers, String header) {
         return allMatchingHeaders(headers, header).findFirst();
     }
+
+    public static Set<String> singleHeaders() {
```

Comment:
Is this needed?

Reply:
Not really needed, I just needed to able access the list from the tests, but I can just copy them there.